CONTRIBUTION.md

```diff
@@ -0,0 +1,83 @@
+# Contribution To The Project
+
+Pull Requests and Issues have four possible Labels:
+* WIP - Work In Progress, it is not ready for review/merge
+* Functional - Proposed change changes the test behaviour
+* Metadata - Only Metadata is about to change
+* Scope - Big scope, a commit changes multiple test scenarios
+
+Rules:
+* A commit introduces one logical unit and it should not alter multiple testcases
```

Comment:
What is a logical unit? A file? In my opinion it would make more sense to mention a _logical change_ (and this may require changes to multiple test cases, in unusual situations)

Reply:
It should be fine now.